studio/app/optinist/core/edit_ROI/wrappers/caiman_edit_roi/merge_roi.py

```diff
@@ -30,6 +30,8 @@ def execute_merge_roi(node_dirpath: str, ids: list):
     num_rois = im.shape[0]
     for i in range(num_rois):
         cell_roi[i, :, :] = np.where(im[i, :, :] != 0, i + 1, np.nan)
+    cell_roi -= 1
+
     merge_roi.append(float(num_rois))
     merge_roi += [(id + 1) for id in ids]
```

Comment:
could you check roi edition is correct?
merge roi and delete roi seem to have problem.

- Note: I've made a commit related to ROI edition(#51 ), so please check after rebase to the latest develop-main.

I've checked with following workflow.
![Screenshot 2023-08-21 at 20 18 58](https://github.com/arayabrain/barebone-studio/assets/42664619/da1dc1f0-b8ed-449f-ab01-0bae11995c71)

After run.
![Screenshot 2023-08-21 at 20 22 26](https://github.com/arayabrain/barebone-studio/assets/42664619/c98b7c0f-e0f2-4743-a866-e978264c217c)

After add roi.
![Screenshot 2023-08-21 at 20 22 56](https://github.com/arayabrain/barebone-studio/assets/42664619/a87fb48c-8b87-44f6-b94e-f92b38814ed4)


- Problem1: In the editor, we can't select id 0.

- Problem2: tried to merge id 2, 3, but 1, 2 merged
![Screenshot 2023-08-21 at 20 23 12](https://github.com/arayabrain/barebone-studio/assets/42664619/1c0f604c-02ea-437b-9d66-109d7159c10e)
![Screenshot 2023-08-21 at 20 23 36](https://github.com/arayabrain/barebone-studio/assets/42664619/9fb9fd94-7817-4686-84b5-0cb90419a0f4)

- Problem3: tried to delete id 7, but 6 deleted
![Screenshot 2023-08-21 at 20 24 14](https://github.com/arayabrain/barebone-studio/assets/42664619/016769f8-9279-497f-975f-a5a5c0da7679)
![Screenshot 2023-08-21 at 20 24 33](https://github.com/arayabrain/barebone-studio/assets/42664619/0b1eccae-7c98-4e13-91fc-2920bf4f2296)

Reply:
@ReiHashimoto 
With the current modification, when merging or deleting ROI roi ID list `[roi1, roi2, roi3, ...]`, FE is sending to server `[roi1-1, roi2-1, roi3-1, ...]` which leads to incorrect ROI identification.
I think the client side also needs adjust to synchronize with the server side and to prevent confusion

Reply:
@quanpython Thank you for your investigation.
I've requested Hung-san to assign FE member.